static/src/inline-svgs/notifications-explainer-desktop.svg

```diff
@@ -1 +1 @@
h fill="#757575" d="M284 48h8v4h-8zM83 57v2h-2v-2h2m1-1h-4v4h4v-4z"/><path fill="#CDB21B" d="M88 56h4v4h-4z"/><path fill="#757575" d="M96 56h20v4H96zM128 56h24v4h-24z"/><path fill="#BCBCBC" d="M156 56h80v4h-80z"/><path fill="#757575" d="M284 56h8v4h-8zM83 65v2h-2v-2h2m1-1h-4v4h4v-4z"/><path fill="#CDB21B" d="M88 64h4v4h-4z"/><path fill="#757575" d="M96 64h20v4H96zM128 64h24v4h-24z"/><path fill="#BCBCBC" d="M156 64h112v4H156z"/><path fill="#757575" d="M284 64h8v4h-8zM83 73v2h-2v-2h2m1-1h-4v4h4v-4z"/><path fill="#CDB21B" d="M88 72h4v4h-4z"/><path fill="#757575" d="M96 72h20v4H96zM128 72h24v4h-24z"/><path fill="#BCBCBC" d="M156 72h80v4h-80z"/><path fill="#757575" d="M284 72h8v4h-8z"/><g><path fill="#757575" d="M83 81v2h-2v-2h2m1-1h-4v4h4v-4z"/></g><g><path fill="#CDB21B" d="M88 80h4v4h-4z"/></g><path fill="#757575" d="M96 80h20v4H96zM128 80h24v4h-24z"/><path fill="#BCBCBC" d="M156 80h100v4H156z"/><path fill="#757575" d="M284 80h8v4h-8z"/><g><path fill="#FFF" d="M236 12h60v12h-60z"/><path fill="#025589" d="M236 12h12v12h-12z"/><path fill="#333" d="M252 13h12v2h-12zM252 16.5h32v1h-32zM252 19h32v1h-32z"/><path fill="#BCBCBC" d="M252 21.5h12v1h-12z"/></g><path fill-rule="evenodd" clip-rule="evenodd" fill="#DCDCDC" d="M312 184V8c0-4.4-3.6-8-8-8H40c-4.4 0-8 3.6-8 8v176H0v8c6.6 4.3 20 4 20 4h304c12.8-.1 20-4 20-4v-8h-32zm-8-8H40V8h264v168z"/><g><path fill="#757575" d="M67 117v2h-2v-2h2m1-1h-4v4h4v-4z"/></g><g><path fill="#FFF" d="M188 88h388v108H188z"/><defs><path id="p" d="M566.5 105l-8.5-8.5 1.5-1.5 8.5 8.5-1.5 1.5zm0 0"/></defs><use xlink:href="#p" overflow="visible" fill-rule="evenodd" clip-rule="evenodd" fill="#767676"/><path fill="#025589" d="M80 88h108v108H80z"/><defs><path id="q" d="M568 96.5l-8.5 8.5-1.5-1.5 8.5-8.5 1.5 1.5zm0 0"/></defs><use xlink:href="#q" overflow="visible" fill-rule="evenodd" clip-rule="evenodd" fill="#767676"/><path fill="#FFF" d="M118.9 107.9c-4 2.9-6.5 7.4-6.5 12.9 0 6.5 3.4 10.9 8.2 14.1l.2.2c-2.8 1.9-8.5 6.4-8.5 11.5 0 3.9 2.5 7.6 7.9 8.8-5.6 1.1-11.9 3.6-11.9 9.9v.5c-6.3-6.5-10.4-15.2-10.4-25.1.1-14.7 8.9-27 21-32.8zm28.9 56.1c0 3.9-2.9 6.9-13.5 6.9-9.3 0-12.5-3.2-12.5-6.7 0-2.6 1.1-5.8 5.7-5.8h13.3c4.6-.1 7 2.2 7 5.6zm11.4 2.3c1.2-2.5 2-5.4 2-8.6 0-9.8-5.2-14.1-16.3-14.1h-17.8c-1.9 0-3.5-1.6-3.5-3.3 0-1.2 1-2.6 2.3-3.6 2.6.9 4.9 1 8.1 1 12.7 0 21.4-6.2 21.3-17-.1-6.1-2.7-10.5-7-13.1 12.7 5.5 21.8 18.1 21.8 33.1-.1 10-4.3 19-10.9 25.6zm-18.7-45.5c0 9.8-2.9 10.9-6.5 10.9-3.5 0-6.5-.8-6.5-10.9s2.9-11.4 6.5-11.4c3.7 0 6.5 1.6 6.5 11.4z"/><text transform="translate(212 139.953)" fill="#333" font-family="'TS3TextSans'" font-size="16">Obama plan for trade deals with Asia and</text><text transform="translate(212 157.953)" fill="#333" font-size="16">Europe in tatters after Senate vote</text><text transform="translate(212 115.953)" fill="#333" font-family="'TS3TextSans'" font-size="20">The Guardian</text><text transform="translate(212 180)" fill="#BCBCBC" font-family="'TS3TextSans'" font-size="14">theguardian.com</text></g></svg>
```

Comment:
Looks like you need to run this through SVGO? Also need to remove the `xmlns` attributes.